Libraries/Opc.Ua.Server/Server/StandardServer.cs

```diff
@@ -2155,7 +2155,8 @@ public ServerStatusDataType GetStatus()
         /// <returns>Boolean value.</returns>
         public async Task<bool> RegisterWithDiscoveryServer()
         {
-            ApplicationConfiguration configuration = string.IsNullOrEmpty(base.Configuration.SourceFilePath) ? base.Configuration : await ApplicationConfiguration.Load(new FileInfo(base.Configuration.SourceFilePath), ApplicationType.Server, null, false);
+            ApplicationConfiguration configuration = string.IsNullOrEmpty(base.Configuration.SourceFilePath) ?
+                base.Configuration : await ApplicationConfiguration.Load(new FileInfo(base.Configuration.SourceFilePath), ApplicationType.Server, null, false);
```

Comment:
Consider adding parameter names to null and false

Reply:
again just code style... better in a seperate PR.